unlock-protocol.com/src/components/content/HomeContent.js

```diff
@@ -49,98 +53,120 @@ export const HomeContent = () => {
         <OpenGraphTags />
       </Head>
 
-      <Hero>
-        <NoPhone>
-          <img alt="Unlock" src="/static/images/pages/svg/hero.svg" />
-        </NoPhone>
-        <Actions>
-          <h1>Monetize the Web with memberships</h1>
-          <Buttons>
-            <HomepageButton
-              label="Dashboard"
-              destination={`${
-                getConfig().publicRuntimeConfig.unlockApp
-              }/dashboard`}
+      <Grid>
+        <Hero>Monetize the web through Memberships</Hero>
+        <BannerKey
+          src="/static/images/illustrations/banner-key.png"
+          alt=""
+          width="192"
+        />
+        <SubTitle>
+          Unlock lets you let’s you easily lock and manage access to your
```

Comment:
Phoneposting so I can’t use the suggestion feature, but you have “lets you” twice on this line

Reply:
Dang. that's rigth! Thanks!